corehq/apps/export/models/new.py

```diff
@@ -230,42 +289,70 @@ def _generate_schema_from_xform(xform, langs, appVersion):
 
         return schema
 
-    @staticmethod
-    def _merge_schema(schema1, schema2):
-        """Merges two ExportDataSchemas together
 
-        :param schema1: The first ExportDataSchema
-        :param schema2: The second ExportDataSchema
-        :returns: The merged ExportDataSchema
-        """
+class CaseExportDataSchema(ExportDataSchema):
 
-        schema = ExportDataSchema()
+    @staticmethod
+    def generate_schema_from_builds(domain, app_id, case_type):
```

Comment:
huh yeah that makes now that i think of it. the reason i thought this though is that in the UI we force the user to select an application before creating an export. that also has ramifications for this strategy. it means we'd have to go through every app and every app build for that case type